evm/its.js

```diff
@@ -696,85 +718,265 @@ async function main(options) {
-    program.addOption(new Option('--sender <sender>', 'TokenManager deployer address'));
-    program.addOption(
-        new Option('--type <type>', 'TokenManager implementation type').choices([
-            'MINT_BURN',
-            'MINT_BURN_FROM',
-            'LOCK_UNLOCK',
-            'LOCK_UNLOCK_FEE',
-        ]),
-    );
-    program.addOption(new Option('--destinationChain <destinationChain>', 'destination chain'));
-    program.addOption(new Option('--destinationAddress <destinationAddress>', 'destination address'));
-    program.addOption(new Option('--params <params>', 'params for TokenManager deployment'));
-    program.addOption(new Option('--tokenAddress <tokenAddress>', 'token address to use for token manager deployment'));
-    program.addOption(
-        new Option(
-            '--destinationTokenAddress <destinationTokenAddress>',
-            'token address on the destination chain to link with the token on the source chain corresponding to the tokenId',
-        ),
-    );
-    program.addOption(new Option('--operator <operator>', 'operator address to use for token manager'));
-    program.addOption(new Option('--gasValue <gasValue>', 'gas value').default(0));
-    program.addOption(new Option('--name <name>', 'token name'));
-    program.addOption(new Option('--symbol <symbol>', 'token symbol'));
-    program.addOption(new Option('--decimals <decimals>', 'token decimals'));
-    program.addOption(new Option('--minter <minter>', 'token minter').default(AddressZero));
-    program.addOption(new Option('--sourceChain <sourceChain>', 'source chain'));
-    program.addOption(new Option('--sourceAddress <sourceAddress>', 'source address'));
-    program.addOption(new Option('--payload <payload>', 'payload'));
-    program.addOption(new Option('--amount <amount>', 'token amount, in terms of symbol'));
-    program.addOption(new Option('--metadata <metadata>', 'token transfer metadata').default('0x'));
-    program.addOption(new Option('--data <data>', 'token transfer data'));
-    program.addOption(new Option('--tokenIds <tokenIds>', 'tokenId array'));
-    program.addOption(new Option('--flowLimits <flowLimits>', 'flow limit array'));
-    program.addOption(new Option('--trustedChain <trustedChain>', 'chain name for trusted addresses'));
-    program.addOption(new Option('--trustedAddress <trustedAddress>', 'trusted address'));
-    program.addOption(new Option('--pauseStatus <pauseStatus>', 'pause status').choices(['true', 'false']));
-    program.addOption(new Option('--rawSalt <rawSalt>', 'raw deployment salt').env('RAW_SALT'));
-
-    program.action((options) => {
-        main(options);
-    });
+    const contractIdCmd = new Command()
+        .name('contract-id')
+        .description('Get contract ID')
+        .action((options) => {
+            options.action = 'contract-id';
+            options.args = [];
+            main(options);
+        });
```

Comment:
```suggestion
        .action((options, cmd) => {
            main(cmd.name(), [], options);
        });
```
avoid half-baked refactors. If the old approach doesn't make sense after a refactor, then change it